packages/loader/container-loader/src/container.ts

```diff
@@ -1584,6 +1584,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         });
 
         deltaManager.on("readonly", (readonly) => {
+            this.context.setConnectionState(!readonly, this.clientId);
```

Comment:
This does not look right. What if we are disconnected & !readonly? we will tell runtime we are connected?